### ISSUE TITLE
shadow-v0 gives error in IE11

### ISSUE THREAD

**Issue by roelfjan** [ampproject/amphtml#8025]
shadow-v0 gives an error in IE11, because the file uses [`ChildNode.remove()`](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove), which isn't supported in IE11.

Error:
```
SCRIPT438: Object doesn't support property or method 'remove'
File: shadow-v0.js, Line: 16, Column: 208
```

Used url [https://cdn.ampproject.org/shadow-v0.js](https://cdn.ampproject.org/shadow-v0.js)

## What browsers are affected?
Internet Explorer 11 (and lower)

cc: @andreban

**Comment by choumx** [comment]
Do we expect `shadow-v0.js` to support IE11 to begin with considering [Shadow DOM isn't supported on the browser](http://caniuse.com/#feat=shadowdom)? 

/cc @dvoytenko

**Comment by dvoytenko** [comment]
Absence of Shadow DOM should be ok. I'll check on remove. We do not allow `Node.remove` in our code, so in either case the presence of it would be wrong.

**Comment by dvoytenko** [comment]
Hmm. Tried on IE11 and Edge - both seem to work. @roelfjan Do you think you can share a demo URL for us to debug?

**Comment by aghassemi** [comment]
Should be resolved by #10129 (we were not really disallowing `remove()` until recently). Closing, please feel free to reopen if still an issue for you.
